### PR TITLE
Fix Number.prototype.toFixed if argument is outside int32 range

### PR DESCRIPTION
**Pull request by dbatyai**
JerryScript-DCO-1.0-Signed-off-by: Dániel Bátyai dbatyai.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by egavrin** [comment]
Good to me

File: jerry-core/ecma/builtin-objects/ecma-builtin-number-prototype.cpp [hunk 1]
```diff
@@ -251,6 +248,9 @@ ecma_builtin_number_prototype_object_to_fixed (ecma_value_t this_arg, /**< this
         int32_t num_digits = 0;
         int32_t exponent = 1;
 
+        /* 1. */
+        int32_t frac_digits = ecma_number_to_int32 (arg_num);
+
```
Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtin-number-prototype.cpp]: number_to_int32 returns 0 for -0.9? I don't know, just asking.

Comment (dbatyai) [inline jerry-core/ecma/builtin-objects/ecma-builtin-number-prototype.cpp]: It does. I'll add a test case for this.

**Comment by galpeter** [comment]
lgtm

**Pull request merged**